keyboards/royal_kludge/rk61_rgb/config_led.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 euwbah <euwbah@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Backlight configuration
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
+#define RGBLIGHT_HUE_STEP 10
+#define RGBLIGHT_LIMIT_VAL 255
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+
+#define LED_MATRIX_ROWS 5
+#define LED_MATRIX_ROW_CHANNELS 3
+#define LED_MATRIX_ROWS_HW (LED_MATRIX_ROWS * LED_MATRIX_ROW_CHANNELS)
+#define LED_MATRIX_ROW_PINS { C5, C4, C6, C8, C7, C9, C11, C10, C12, C14, C13, B13, B15, D3, B14 }
+
+#define LED_MATRIX_COLS 14
+#define LED_MATRIX_COL_PINS { A8, A9, A10, A11, A12, A13, A14, A15, B0, B1, B2, B3, B4, B5 }
```

Comment:
these are obsolete flags. you can replace with simply this
```suggestion
#define SN32_RGB_MATRIX_ROW_PINS { C5, C4, C6, C8, C7, C9, C11, C10, C12, C14, C13, B13, B15, D3, B14 }
#define RGB_MATRIX_LED_COUNT (70)
```

Reply:
Had problems compiling for RK61 after deleting 

`#define LED_MATRIX_COL_PINS { A8, A9, A10, A11, A12, A13, A14, A15, B0, B1, B2, B3, B4, B5 }`

Strangely, the correspondent line for the RK68 could be deleted with no problem...

Both keyboard files are extremely identical and I can't understand why RK61 doesn't compile without this line and RK68 does... 

Also, I had to move the definition of `SN32_RGB_MATRIX_ROW_PINS` to config.h and remove it from config_led.h.
Finally, I also removed RGB_MATRIX_LED_COUNT as it was conflicting with autogenerated value extracted from info.json.